Packs/Core/Playbooks/playbook-Possible_External_RDP_Brute-Force.yml

```diff
@@ -2509,6 +2509,35 @@ inputs:
   required: false
   description: Set this value to "true" to allow early containment when the IP is known malicious
   playbookInputQuery:
+inputSections:
+- inputs:
+  - SOCEmailAddress
+  - UserEngagementThreshold
+  - EndpointID
+  - Country
+  name: Incident Management
```

Comment:
I will stop adding the alert... please look and fix all the playbook input section (also for the section description)